app/src/main/java/se/tmeit/app/ui/AboutFragment.java

```diff
@@ -30,12 +30,12 @@ public int getTitle() {
 	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
 		View view = inflater.inflate(R.layout.fragment_about, container, false);
 
-		TextView aboutText = (TextView) view.findViewById(R.id.about_text);
+		TextView aboutText = view.findViewById(R.id.about_text);
```

Comment:
Om du ändå fixar lintcheckar så gör alla variabler som inte ändras final också. :)
```suggestion
		final TextView aboutText = view.findViewById(R.id.about_text);
```